software/interface.md

```diff
@@ -90,33 +90,37 @@ You can update DAPLink to this beta version following the microbit.org website [
-2. `interface mode` is used to target the nRF52833 <span class="v2">V2</span> or nRF51822 <span class="v1">V1</span>. In this mode, the drive name is `MICROBIT` and the hex files dropped onto the micro:bit are written to the flash of the target MCU.
+1. `bootloader mode` is used to for updating the interface firmware. In this mode, the drive name is `MAINTENANCE` and hex files dropped onto the disk are written into the interface MCU flash. These files MUST contain an image of DAPLink or equivalent.
+2. `interface mode` is used to flash the target MCU. In this mode, the drive name is `MICROBIT`.
 
 There are detailed instructions for how to [update the firmware version on the micro:bit website](https://microbit.org/get-started/user-guide/firmware/).
 
 **You should never update your micro:bit with firmware from untrusted sources, as these could damage your micro:bit, or make it impossible to re-flash**
 
 ## Files on the MICROBIT drive
 
-The flash file system presented on the micro:bit drive is entirely virtual. It is not backed by real memory, and this is why the drive ejects itself after new files are written. When a file is dropped onto the MICROBIT drive, instead of being written into flash memory (like a normal USB memory stick), it is streamed to the target MCU.
+The flash file system presented on the micro:bit drive is entirely virtual. It is not backed by real memory, and this is why the drive ejects itself after new files are written. When a file is dropped onto the MICROBIT drive, instead of being written into storage memory (like a normal USB memory stick), it is streamed to the target MCU.
 
 The following virtual files exist on the file-system:
 
 - `DETAILS.TXT`: This file tells you about the build of DAPLink currently installed on the interface chip. In later versions of DAPLink it also includes more diagnostic information.
 
 - `MICROBIT.HTM`: This is a link to [microbit.org](https://microbit.org) to help you get started.
 
+- `MY_DATA.HTM` : This is a [data logging](https://microbit.org/get-started/user-guide/data-logging/) file and might not be always present in the MICROBIT drive.
+
 After flashing occurs, there might also be a `FAIL.TXT` file, that gives a cause for failure.
 
 The file [error.c](https://github.com/mbedmicro/DAPLink/blob/master/source/daplink/error.c) in the DAPLink source can help in diagnosing what these errors mean.
 
 There is also a full list of [micro:bit error codes](https://support.microbit.org/en/support/solutions/articles/19000016969-micro-bit-error-codes) in our Knowledgebase.
 
+An `ASSERT.TXT` file might also appears, this indicates a minor error might have occurred, but DAPLink was able to recover and work as expected.
```

Comment:
```suggestion
An `ASSERT.TXT` file might also appear, this indicates a minor error might have occurred, but DAPLink was able to recover and work as expected.
```

Reply:
Added in force push: https://github.com/microbit-foundation/dev-docs/compare/b96c9a28afef09f7aee657bd4f43ec641afa0d3a..400811505ff84b6d6402dc6ccbd92c1283666b27